packages/trader/src/Modules/Trading/Containers/purchase.tsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import { isAccumulatorContract, isEmptyObject, isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import PurchaseButtonsOverlay from 'Modules/Trading/Components/Elements/purchase-buttons-overlay.jsx';
-import PurchaseFieldset from 'Modules/Trading/Components/Elements/purchase-fieldset.jsx';
+import PurchaseFieldset from 'Modules/Trading/Components/Elements/purchase-fieldset';
 import { getContractTypePosition } from 'Constants/contract';
 import { useTraderStore } from 'Stores/useTraderStores';
 import { observer, useStore } from '@deriv/stores';
+import { TProposalTypeInfo } from '../../../../types';
```

Comment:
@kate-deriv let's add ALIAS for the types file in packages/trader/build/constants.js like it's done e.g. for Stores